virttest/remote_commander/remote_runner.py

```diff
@@ -544,6 +548,10 @@ def cmd_loop(self):
                     cmd = CmdSlave(self.read_msg()[1])
                     self.cmds[cmd.cmd_id] = cmd
                     try:
+                        # There is hidden bug. We can bump into condition when
+                        # we running some function-command. That function dump
+                        # some data to stdio. But, stdio is out of
+                        # consideration at this point. It can stuck.
```

Comment:
You mean the pipe get's filled and the process is stalled? Can we disable the stdout, or add thread which reads-it-out?